index.html

```diff
@@ -7,5 +7,6 @@
 </head>
 <body>
     <h1>Demo</h1>
+    <h1>Demo</h1>
```

Comment:
Remove duplicate code
Add script tag
Check this link: https://www.w3schools.com/js/js_whereto.asp

Reply:
Done